packages/algob/src/types.ts

```diff
@@ -34,6 +34,11 @@ export type AccountDef =
   | HDAccount
   | Account;
 
+export interface wallet {
```

Comment:
Change `wallet` to `Wallet`, it's a type

Reply:
ok